Client/Frontend/Home/JumpBackIn/JumpBackInSectionLayout.swift

```diff
@@ -4,9 +4,9 @@
 
 import Foundation
 
-enum JumpBackInDisplayGroup {
-    case jumpBackIn
-    case syncedTab
+struct JumpBackInDisplayGroupCount {
+    var tabsCount: Int
+    var syncTabCount: Int
```

Comment:
`syncTabCount` > `syncedTabCount`? syncTabCount sounds weird to me.

Reply:
Also with the change from enum to struct we should change the tests to check for the number of syncTabCount/syncedTabCount as well.